buildSrc/src/main/kotlin/BuildInfo.kt

```diff
@@ -20,47 +18,25 @@ open class BuildInfo(project: Project) {
       libs.findVersion("graalVm").get().toString()
     }
 
-    val isGraal22: Boolean by lazy {
-      version.startsWith("22")
-    }
-
-    val arch by lazy {
-      if (os.isMacOsX && isGraal22) {
-        "amd64"
-      } else {
-        self.arch
-      }
+    val graalVmJavaVersion: String by lazy {
+      libs.findVersion("graalVMJdkVersion").get().toString()
```

Comment:
nit: `graalVmJavaVersion` vs. `graalVMJdkVersion` (two differences)

Reply:
Yeah, fair. Going to split the difference here (`graalVmJdkVersion`)